common/src/main/java/com/viaversion/viarewind/protocol/protocol1_8to1_9/cooldown/BossBarVisualization.java

```diff
@@ -0,0 +1,49 @@
+package com.viaversion.viarewind.protocol.protocol1_8to1_9.cooldown;
+
+import com.viaversion.viarewind.protocol.protocol1_8to1_9.Protocol1_8To1_9;
+import com.viaversion.viarewind.utils.PacketUtil;
+import com.viaversion.viaversion.api.connection.UserConnection;
+import com.viaversion.viaversion.api.protocol.packet.PacketWrapper;
+import com.viaversion.viaversion.api.type.Type;
+import com.viaversion.viaversion.libs.gson.JsonPrimitive;
+import com.viaversion.viaversion.protocols.protocol1_9to1_8.ClientboundPackets1_9;
+import java.util.UUID;
+
+public class BossBarVisualization implements CooldownVisualization {
+	private final UserConnection user;
+	private UUID bossUUID;
+
+	public BossBarVisualization(UserConnection user) {
+		this.user = user;
+	}
+
+	@Override
+	public void show(double progress) {
+		PacketWrapper wrapper = PacketWrapper.create(ClientboundPackets1_9.BOSSBAR, user);
+		if (bossUUID == null) {
+			bossUUID = UUID.randomUUID();
+			wrapper.write(Type.UUID, bossUUID);
```

Comment:
Please add comments to all packets you're sending to explain it's content, e. g.

wrapper.write(Type.VAR_INT, 0); // entity id